lib/src/main/java/com/thenewboston/data/dto/bankapi/upgradenoticedto/UpgradeNoticeRequest.kt

```diff
@@ -16,5 +16,5 @@ data class UpgradeNoticeRequest(
 @Serializable
 data class UpgradeNoticeMessage(
     @SerialName("bank_node_identifier")
-    val bankNodeIdentifier: String
+    val otherNodeIdentifier: String
```

Comment:
What's the reason for this rename?

Reply:
Instead of repeating ourselves just create a nullable variable for both bank node and validator node strings.